### PR TITLE
Updated Movie Entity revenue and budget field from int to long. & Test Fixes.

### PR DESCRIPTION
**Pull request by ProIcons**
- Updated Movie Entity revenue and budget field from int to long. & Test Fixes.
- Updated Company entity added origin_country field & Updated Tests
- Upgrade to latest retrofit release to resolve conflicts for projects using newer version of okhttp.

### CODE REVIEW

File: src/test/java/com/uwetrottmann/tmdb2/assertions/CompanyAssertions.java [hunk 1]
```diff
@@ -34,7 +34,8 @@ public static void assertCompany(Company company) {
 
         assertThat(company.headquarters).isNotNull();
         assertThat(company.homepage).isNotNull();
-
+        assertThat(company.origin_country).isNotEmpty();
+        assertThat(company.origin_country).isNotEmpty();
```
Comment (UweTrottmann) [inline src/test/java/com/uwetrottmann/tmdb2/assertions/CompanyAssertions.java]: Any reason this is asserted twice?

**Comment by UweTrottmann** [comment]
Thanks for this PR! See my comment, once resolved I will merge once I'm next working on this project.

File: src/test/java/com/uwetrottmann/tmdb2/assertions/CompanyAssertions.java (hunk 1 above)
Comment (ProIcons) [reply src/test/java/com/uwetrottmann/tmdb2/assertions/CompanyAssertions.java]: 👍

**Comment by ProIcons** [comment]
Done

**Pull request merged**